spec/models/band_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe Band do
+  describe 'associations' do
+    it { should have_many(:musicians) }
```

Comment:
because Macho approves of the should syntax, and he's the one who helped me out while setting up the DB and first tests